var/spack/repos/builtin/packages/opencv/package.py

```diff
@@ -254,8 +260,8 @@ class Opencv(CMakePackage, CudaPackage):
         depends_on("python@3.2:", type=("build", "link", "run"))
         depends_on("py-setuptools", type="build")
         depends_on("py-numpy", type=("build", "run"))
-        # https://github.com/opencv/opencv-python/issues/943
-        depends_on("py-numpy@:1", when="@:4.10.0.83", type=("build", "run"))
+        # https://github.com/opencv/opencv/issues/25455
+        depends_on("py-numpy@:1", when="@:4.9", type=("build", "run"))
```

Comment:
To clarify this change, opencv 4.10.0 builds fine with numpy 2 from source, but the opencv-python repo was still building with numpy 1 when uploading wheels until 4.10.0.84. So we can move this bound back a bit.